crates/nu-command/src/conversions/into/datetime.rs

```diff
@@ -67,6 +67,7 @@ impl Command for SubCommand {
         .input_output_types(vec![
             (Type::Int, Type::Date),
             (Type::String, Type::Date),
+            (Type::List(Box::new(Type::String)), Type::List(Box::new(Type::Date))),
```

Comment:
Would [`.vectorizes_over_list()`](https://docs.rs/nu-protocol/latest/nu_protocol/struct.Signature.html#method.vectorizes_over_list) suffice here? Or does it not work for the `Int -> Date` signature like that.

Reply:
I remove that new i/o line and added `.vectorizes_over_list()` and still get the same error.
```nushell
seq date -r --days 7 | into datetime
Error: nu::parser::input_type_mismatch

  × Command does not support list<string> input.
   ╭─[entry #1:1:1]
 1 │ seq date -r --days 7 | into datetime
   ·                        ──────┬──────
   ·                              ╰── command doesn't support list<string> input
```